app/controllers/orders_controller.rb

```diff
@@ -1,5 +1,9 @@
 class OrdersController <ApplicationController
 
+  def index
+    @orders = Order.all
```

Comment:
This will work but the orders are for all users. I think the order should be specific to the user who has logged in. I was thinking something like.
`current_user.orders`
This will give the orders for a specific person.

Reply:
Yeah good call! that makes more sense